src/pages/list/CardListPage.jsx

```diff
@@ -103,12 +88,25 @@ function CardListPage() {
     ),
     prevArrow: <CustomPrevArrow currentSlide={currentSlide} />,
   };
+  const hotsettings = {
+    dots: false,
+    infinite: false,
+    speed: 500,
+    slidesToShow: slidesToShow,
+    slidesToScroll: 2,
+    initialSlide: 0,
+    afterChange: (current) => setHotCurrentSlide(current),
+    nextArrow: (
+      <CustomNextArrow currentSlide={hotCurrentSlide} slideCount={limit} />
+    ),
+    prevArrow: <CustomPrevArrow currentSlide={hotCurrentSlide} />,
+  };
 
   function CustomNextArrow(props) {
     const { className, style, onClick } = props;
```

Comment:
props를 받아오고 구조분해하지말고 받아올때 구조분해해서 받으면 좋을거같아용~
```suggestion
function CustomNextArrow({className, style, onClick})
```

Reply:
수정 반영했습니다